coordinator/src/main/java/org/apache/uniffle/coordinator/web/resource/ApplicationResource.java

```diff
@@ -46,9 +45,12 @@ public class ApplicationResource extends BaseResource {
   public Response<Map<String, Integer>> getAppTotality() {
     return execute(
         () -> {
-          Set<String> appIds = getApplicationManager().getAppIds();
           Map<String, Integer> appTotalityMap = new HashMap<>();
-          appTotalityMap.put("appTotality", appIds.size());
+          if (getApplicationManager().getQuotaManager() != null) {
+            appTotalityMap.put(
+                "appTotality",
+                getApplicationManager().getQuotaManager().getAppIdToUser().keySet().size());
```

Comment:
Is `getApplicationManager().getAppIds().size()` better?

Reply:
I think all interfaces of ApplicationResource have the same problems.

Reply:
The following code have the same problem. @yl09099 
https://github.com/apache/incubator-uniffle/blob/800bf159479e96c6f1911f827e818d3de71e8b51/coordinator/src/main/java/org/apache/uniffle/coordinator/web/resource/ApplicationResource.java#L62
https://github.com/apache/incubator-uniffle/blob/800bf159479e96c6f1911f827e818d3de71e8b51/coordinator/src/main/java/org/apache/uniffle/coordinator/web/resource/ApplicationResource.java#L81-L83

Reply:
> The following code have the same problem. @yl09099
> 
> https://github.com/apache/incubator-uniffle/blob/800bf159479e96c6f1911f827e818d3de71e8b51/coordinator/src/main/java/org/apache/uniffle/coordinator/web/resource/ApplicationResource.java#L62
> 
> 
> https://github.com/apache/incubator-uniffle/blob/800bf159479e96c6f1911f827e818d3de71e8b51/coordinator/src/main/java/org/apache/uniffle/coordinator/web/resource/ApplicationResource.java#L81-L83

Have been modified.